drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/SettingsViewImpl.html

```diff
@@ -0,0 +1,96 @@
+                Type
+            </label>
+            <span data-field="scenarioType" id="kieScenarioSettingsType"></span>
+        </div>
+        <br/>
+        <div data-field="ruleSettings">
+            <div class="pf-c-form__group">
+                <label for="kieScenarioSettingsKieSession">
+                    KieSession
+                </label>
+                <input class="form-control" data-field="kieSession" id="kieScenarioSettingsKieSession">
+            </div>
+            <br/>
+            <div class="pf-c-form__group">
+                <label for="kieScenarioSettingsKieBase">
+                    KieBase
+                </label>
+                <input class="form-control" data-field="kieBase" id="kieScenarioSettingsKieBase">
+            </div>
+            <br/>
+            <div class="pf-c-form__group">
+                <label for="kieScenarioSettingsRuleFlowGroup">
+                    RuleFlowGroup
+                </label>
+                <input class="form-control" data-field="ruleFlowGroup" id="kieScenarioSettingsRuleFlowGroup">
+            </div>
+            <br/>
+            <div class="pf-c-form__group">
+                <label for="kieScenarioSettingsDMOSession">
+                    DMOSession
+                </label>
+                <input class="form-control" data-field="dmoSession" id="kieScenarioSettingsDMOSession">
+            </div>
+        </div>
+        <div data-field="dmnSettings">
+            <div class="pf-c-form__group">
+                <label for="kieScenarioSettingsModelPath" data-field="dmnFileLabel">
+                    DMN model
+                </label>
+                <span class="form-control" data-field="dmnFilePath" id="kieScenarioSettingsModelPath"></span>
+            </div>
+            <br/>
+            <div class="pf-c-form__group">
+                <label for="kieScenarioSettingsDMNNamespace" data-field="dmnNamespaceLabel">
+                    DMN namespace
+                </label>
+                <span class="form-control" data-field="dmnNamespace" id="kieScenarioSettingsDMNNamespace"></span>
+            </div>
+            <br/>
+            <div class="pf-c-form__group">
+                <label for="kieScenarioSettingsDMNName" data-field="dmnNameLabel">
+                    DMN name
+                </label>
+                <span class="form-control" data-field="dmnName" id="kieScenarioSettingsDMNName"></span>
+            </div>
+        </div>
+        <br/>
+        <div class="checkbox">
+            <label for="kieScenarioSettingsSkipFromBuild">
+                <input type="checkbox" data-field="skipFromBuild" id="kieScenarioSettingsSkipFromBuild">Skip this simulation during the build
+            </label>
+        </div>
+        <br/>
+        <p data-field="toBeRemoved" style="font-style: italic">Only the default values are currently supported. Changing them will not have any impact on running scenario.</p>
```

Comment:
@kkufova 
I put that as it is because saving those values will actually change the file scesim - so the asset. It does not effect hot the scesim is "*run*, but it is a different aspect.

Reply:
@gitgabrio, OK, in that case, please change it to:

> Only the default values are currently supported. Changing them will not have any impact on test execution.